frontend/src/common/mock/signup/signup.ts

```diff
@@ -0,0 +1,12 @@
+import { http, HttpResponse } from 'msw';
+
+import { API_ENDPOINTS } from '../../constants/apiEndpoints';
+
+const BASE_URL = process.env.BASE_URL;
+const SIGNUP_URL = `${BASE_URL}${API_ENDPOINTS.SIGNUP}`;
+
+const postSignup = http.post(SIGNUP_URL, async () => {
+  return HttpResponse.json({ message: '회원가입 성공' }, { status: 201 });
```

Comment:
p5: 여기도 await 붙어도 좋을 거 같아 ㅎㅎ

```suggestion
  return await HttpResponse.json({ message: '회원가입 성공' }, { status: 201 });
```

Reply:
[refactor: Promise 타입 return하는 곳에 await 추가](https://github.com/woowacourse-teams/2025-Fit-toring/pull/274/commits/9fd7ebe904db1c0a4bc54716699fede13da986c6)